queries/dev/run.py

```
@@ -1,11 +1,43 @@
 
 # Veja os parâmetros disponíveis da função run_dbt_model em util.py
 
-run_dbt_model(
-    dataset_id="example",
-    table_id="my_first_dbt_model",
+inicio = "2025-04-01"
+fim = "2026-03-15"
+run_dates = pd.date_range(start=inicio, end=fim, freq="D")
+run_dates_list = run_dates.strftime("%Y-%m-%d").tolist()
+partitions = ", ".join([f"date({dt.year}, {dt.month}, {dt.day})" for dt in run_dates])
+vars = {
```

Ruff (0.15.9)

[error] Variable `vars` is shadowing a Python builtin

(A001)